# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxUserFontSet.h
# Commit: 478ac1179fa6
# Full Hash: 478ac1179fa606aa4de5cdbf6a8a11a377e139bb
# Author: Cameron McCormack <cam@mcc.id.au>
# Date: 2019-06-07 09:47:19
# Regressor Bug: 1490792
# File Overlap Count: 1
# Description:
#   Bug 1490792 - Part 1: Move font type determination into SanitizeOpenTypeData. r=jfkthame
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D33894
# ==============================================================================

diff -r 91643c6f9a42 -r 478ac1179fa6 gfx/thebes/gfxUserFontSet.h
--- a/gfx/thebes/gfxUserFontSet.h	Thu Jun 06 19:18:16 2019 +0000
+++ b/gfx/thebes/gfxUserFontSet.h	Thu Jun 06 18:23:27 2019 +0000
@@ -638,7 +638,7 @@
  protected:
   const uint8_t* SanitizeOpenTypeData(const uint8_t* aData, uint32_t aLength,
                                       uint32_t& aSaneLength,
-                                      gfxUserFontType aFontType);
+                                      gfxUserFontType& aFontType);
 
   // attempt to load the next resource in the src list.
   void LoadNextSrc();
